### PR TITLE
Add compatibility with n64split's func_XXXXXXXX: notation

### PR DESCRIPTION
**Pull request by onlymx13**
So you can use mips_to_c on functions in files obtained from n64split.

### CODE REVIEW

File: src/parse_file.py [hunk 1]
```diff
@@ -156,6 +156,13 @@ def parse_file(f: typing.TextIO, options: Options) -> MIPSFile:
                         mips_file.new_jumptable_label(function_name)
                     else:
                         mips_file.new_function(function_name)
+                elif line.startswith('func'):
+                    # Other kind of function label.
+                    function_name: str = line[:-1]
```
Comment (matt-kempster) [inline src/parse_file.py]: I assume this `[:-1]` is to trim off the colon. Can we use `.rstrip()` here instead?

File: src/parse_file.py [hunk 2]
```diff
@@ -156,6 +156,13 @@ def parse_file(f: typing.TextIO, options: Options) -> MIPSFile:
...
+                    if re.match('L(_U_)?[0-9A-F]{8}', function_name):
```
Comment (matt-kempster) [inline src/parse_file.py]: Hmm, IMO too much hardcoding here. Maybe we should just take `L_` as a required suffix and leave it at that?

Comment (onlymx13) [reply src/parse_file.py]: I just copied this from above, I'm not sure what this is checking for.

Comment (simonlindholm) [reply src/parse_file.py]: I agree it's a bit much hard-coding, I'd be in favor of changing both occurrences to checking just that it starts with `L`. (Eventually we'll get rid of that magic as well, but that's a harder problem. `L` *is* well-established as meaning a local label, at least.)

Comment (simonlindholm) [reply src/parse_file.py]: On second thought, this particular check is pretty silly. It checks whether the line stars with "L", after we have already established that it starts with "func".

Comment (onlymx13) [reply src/parse_file.py]: You're right, should I just assume that the statement is always false and remove the check?

Comment (onlymx13) [reply src/parse_file.py]: like what I now have?

Comment (onlymx13) [reply src/parse_file.py]: like what I now have?

Comment (simonlindholm) [reply src/parse_file.py]: yep, looks good! maybe add `and line.endswith(":")` to the if condition?

also, you have a `'` while our coding style says `"` -- running `black` will fix that.

**Pull request merged**